components/common/Layout/Layout.tsx

```diff
@@ -69,12 +70,14 @@ const ModalUI: FC = () => {
   ) : null
 }
 
-const SidebarView: FC<{ sidebarView: string; closeSidebar(): any }> = ({
-  sidebarView,
-  closeSidebar,
-}) => {
+const SidebarView: FC<{
+  sidebarView: string
+  closeSidebar(): any
+  links: any
```

Comment:
If its possible we can create a `Link` interface just on top, something like:

```ts
interface Link{
  href: string
  label: string
}
```

And do `links: Link[]`